packages/features/bookings/Booker/Booker.tsx

```diff
@@ -56,6 +56,11 @@ const BookerComponent = ({
     (state) => [state.selectedTimeslot, state.setSelectedTimeslot],
     shallow
   );
+  // const seatedEventData = useBookerStore((state) => state.seatedEventData);
+  const [seatedEventData, setSeatedEventData] = useBookerStore(
```

Comment:
I think we are not initializing `seatedEventData` here https://github.com/calcom/cal.com/blob/f405ae5824bc0e0750b543ca2e67433077ca3690/packages/features/bookings/Booker/Booker.tsx#L83

This is the reason, when I am trying to a book a slot with 2 seats. 2 different bookings are being created.

Reply:
How are you testing the PR? It's working properly for me.

In terms of setting `seatedEventData` I set it when a slot is selected because we won't know how many seats are available or the bookingUid on the initial load.

Reply:
There is a time when you access directly to a booking form without seatedEventData in store, this is because you didn't select a timeslot which saves slot data to it.
I made some changes that when SSR loading bookingForm, bookingData should retrieve bookingAttendees length so we can use that for that scenario.

Reply:
@alannnc @joeauyeung Can I test it again now?